tests/test-runner/bin/zts-report.py.in

```diff
@@ -279,6 +286,7 @@ if sys.platform.startswith('freebsd'):
         'cli_root/zpool_import/zpool_import_features_003_pos':
             ['FAIL', '11854'],
         'delegate/zfs_allow_003_pos': ['FAIL', known_reason],
+        'fallocate/fallocate_punch-hole': ['FAIL', fspacectl_reason],
```

Comment:
It would be good if we could make it ignore errors only on really uncapable platforms, otherwise we may miss if it break one day.